webapp/app/modules/rooms/rooms.redux.js

```diff
@@ -0,0 +1,45 @@
+import { Record, Map } from 'immutable';
+import { createActions, createReducer } from '../utils/entityRegistry';
+
+
+export const DEFAULT_ROOM = 'main';
+export const MESSAGE_FORM = 'messageForm';
+export const CREATE_ROOM_FORM = 'createRoomForm';
+
+export const {
+  Types: RoomsTypes,
+  Creators: RoomsActions,
+} = createActions({
+  setActiveRoomId: ['id'],
+  createMessage: ['author', 'content'],
+  createRoom: ['name'],
+  openCreateRoomDialog: null,
```

Comment:
czy ten dialog jest czymś co wychodzi poza komponent room? Czy jest sens trzymać to w reduxie? Może wystarczyłoby to zostawić jako state?

Reply:
Zgadzam się, że w tym momencie state całkowicie by wystarczył do tego. Wole jednak zakładać, że tego typu akcje mogą być wywoływane z różnych miejsc aplikacji i dodanie ich do reduxa daje nam zawsze tą możliwość.

Reply:
👍